### DISCUSSION TITLE
466

### DISCUSSION THREAD

**Discussion by giscus** [Shane-Park/markdownBlog#167] [Comments]
# 466

문제 스프링 부트 3.0.0 버전으로 개인 프로젝트를 진행하고 있는데, 시간도 어느정도 흘렀고 해서 3.0.5 버전으로 버전을 높이던 중 MySQL 드라이버를 찾지 못하는 문제가 발생. Failed to load driver class com.mysql.cj.jdbc.Driver in either of HikariConfig class loader or Thread context classloader비교적 최근의 3.0.5 에서의 문제인가 하여 3.0.4, 3.0.3 조금씩 낮춰보며 확인을 해 보았지만 3.0.0 으로 바꾸기 전까지는 해결된 기미가 보이지 않음 원인 원인을 찾아보자. 일단 지금 상태의 build.gradle.kts 파일이다. ${code:build.gradle.kts} import org..

https://shanepark.tistory.com/466

**Comment by bodguy** [comment]
감사합니다~ spring boot 3.x 대로 올라오면서 여러가지 변경점들 때문에 골치아팠는데 덕분에 해결되었습니다~!

**Comment by chhs2131** [comment]
감사합니다! 생각지도 못했네요 ㅎㅎ
